sample/ziti-echo-server/ziti-echo-server.py

```diff
@@ -13,27 +13,41 @@
 #  limitations under the License.
 
 import sys
+import signal
+import threading
 import openziti
 
+def signal_handler(signum, frame):
+    print("\nCtrl-C detected. Exiting...")
+    sys.exit(0)
 
 def run(ziti_id, service):
     ztx = openziti.load(ziti_id)
     server = ztx.bind(service)
     server.listen()
 
     while True:
+        print('beginning accept')
         conn, peer = server.accept()
         print(f'processing incoming client[{peer}]')
         with conn:
             count = 0
             while True:
+                print('starting receive')
                 data = conn.recv(1024)
                 if not data:
                     print(f'client finished after sending {count} bytes')
                     break
                 count += len(data)
                 conn.sendall(data)
+                print(f'received data from client. returning {count} bytes of data now...')
```

Comment:
`count` is the accumulated value.. is that the intent to print total bytes on every message?

Reply:
yah. thx good catch. i'll change it to len(data)

Reply:
and i'll fix the other print that i copied this from too since that's wrong too :)